nexus-repository-composer-it/src/test/java/org/sonatype/nexus/repository/composer/internal/ComposerClient.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2018-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.composer.internal;
+
+import java.net.URI;
+
+import org.sonatype.nexus.testsuite.testsupport.FormatClientSupport;
+
+import org.apache.http.client.protocol.HttpClientContext;
+import org.apache.http.impl.client.CloseableHttpClient;
+
+public class ComposerClient
+    extends FormatClientSupport
+{
+  public ComposerClient(
+      final CloseableHttpClient httpClient,
```

Comment:
Formatting looks off here, pretty sure you can bring it up a line and align the rest of the params.

Reply:
So strange, even though i have the sonatype codestyle, it was set to 4 spaces. Amazing eye.

In: 5f4baf6